src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -194,43 +194,44 @@ export function getEventContent(event) {
   switch (event.type) {
     case EVENT_TYPES.claim:
       return {
-        title: 'VBA sent the original claim decision to you',
+        title: 'VA sent the original claim decision to you',
         description: '',
         liClass: 'section-complete'
       };
     case EVENT_TYPES.nod:
       return {
-        title: 'VBA received your Notice of Disagreement',
+        title: 'VA received your Notice of Disagreement',
         description: '',
         liClass: 'section-complete'
       };
     case EVENT_TYPES.droHearing:
       return {
-        title: 'Dro Hearing',
+        // TODO: Verify property names these are correct
```

Comment:
This is in fact still relevant: When Chris gets back to me [these questions](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7894#issuecomment-358822837), I'll delete these comments.

...and probably even make sure we're grabbing the information from the right place if we're lucky. :man_shrugging:

Reply:
Might almost be worth just [PLACEHOLDER]ing them instead of guessing the property names / paths but not sure it makes a real difference.

Reply:
Just saw that Chris G mentioned the events won't have any fill-in-the-blanks, so not sure if this comment still applies.

Reply:
Yeah, Chris said the content on these are going to change to not include the dynamic bits. I can [PLACEHOLDER] them, but they'll get changed anyhow. :man_shrugging: